code_benches/run.py

```diff
@@ -105,17 +157,18 @@ def sources(self):
     # == Custom Benchmarks ==
     Program("custom_binarytrees", [16], 2 ** 14),
     Program("custom_function_pointers", [], 2 ** 14),
-    Program("custom_libjpeg", [], 2 ** 15,
-            custom_arguments=["-Wno-incompatible-library-redeclaration", "-Wno-implicit-function-declaration",
-                              "-Wno-shift-negative-value"]),
+    #Program("custom_libjpeg", [], 2 ** 15,
```

Comment:
I assume that this is a WASI-SDK related regression as with line 166 below? If so, I concur with @Others. My suggestion is to open an issue for each of these regressions and add TODO comments that include the relevant issue numbers.